app/views/results/_location.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="filter-form" data-qa="filters__location">
+  <h2 class="govuk-heading-s filter-form__title">
+    Location<span class="govuk-visually-hidden">:</span>
```

Comment:
is this omitted/why

![image](https://user-images.githubusercontent.com/470137/74337510-4ec32100-4d98-11ea-8320-9692bc8d0f96.png)

Reply:
@aliuk2012 Not sure what's going on here, do you have any govuk light to shine on it?

Reply:
@faissalMT thinks it's something to do with screen-reader friendliness, it's in all the other filter partials